### PR TITLE
Setup new Botswana HIE project

### PR DESCRIPTION
**Pull request by ItsMurumba**
## Type of PR (check all applicable)

- [x] 💼 New Package
- [ ] ♻️ Refactor
- [ ] ✨ New Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [x] 🚩 Other

## Description

This PR adds the following base packages for the HIE:

- database-postgres
- fhir-datastore-hapi-fhir
- identity-access-manager-keycloak
- interoperability-layer-openhim
- monitoring
- reverse-proxy-nginx

## Tickets & Related Documents

[Setup new Botswana HIE project](https://app.clickup.com/t/86byj70pk)

## Added code snippets?
- [ ] 👍 Yes
- [x] 🙅 No, because they aren't needed

## Added tests?

- [ ] 👍 Yes
- [x] 🙅 No, because they aren't needed
- [ ] 🙋 No, because I need help

## Added to documentation?

- [x] 📜 readme
- [ ] 📓 general documentation
- [ ] 🙅 no documentation needed

## Are there any post-deployment tasks we need to perform?

- Setup hosted dev environment: To allow for actual testing of different workflows/configurations

### CODE REVIEW

**Review by marrouchi — APPROVED** [review]

File: ,env.hie [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# General
```
Comment (marrouchi) [inline ,env.hie:1]: @ItsMurumba Please rename this file to ".env.hie" (there's a coma)

**Review by marrouchi — CHANGES_REQUESTED** [review]

File: config.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+profiles:
+  - name: dev
+    packages:
+      - interoperability-layer-openhim
+      - database-postgres
+      - fhir-datastore-hapi-fhir
+      - monitoring
+      - identity-access-manager-keycloak
+      - database-postgres
```
Comment (BMartinos) [inline config.yaml:21]: This package seems to be specified already and can be removed.

File: config.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    envFiles:
+      - .env.local
+    dev: true
+    only: true
```
Comment (BMartinos) [inline config.yaml:25]: Question: Do we want to only run the defined packages, which will exclude any dependant packages? I might have this wrong, but i think with this flag, only the packages specified are deployed and their dependencies are ignored

File: config.yaml (hunk 2 above)
Comment (marrouchi) [reply config.yaml:21]: Sorry, which one ?

Comment (BMartinos) [reply config.yaml:21]: The `database-postgres` package is already defined on line 17:
https://github.com/BWHIE/hie-botswana/pull/1/files/25e2ce7b5af48b2c391dad2423891133c3ca5446#diff-d8d0422389f03d783e32e627250fe29834bd09c6361640d1ff00661dd6820034R17

Comment (marrouchi) [reply config.yaml:21]: :D Oh yes 🙏

**Pull request merged**